x/fungible/keeper/v2_deposits.go

```diff
@@ -49,6 +49,7 @@ func (k Keeper) ProcessV2Deposit(
 // ProcessV2RevertDeposit handles a revert deposit from an inbound tx with protocol version 2
 func (k Keeper) ProcessV2RevertDeposit(
 	ctx sdk.Context,
+	sender string,
```

Comment:
In these locations, can we maybe rename sender to `inboundSender` or `initialSender`? To not confuse with the sender if the outbound